README.md

```diff
@@ -1028,13 +1028,60 @@ To Start using the sqlite3 library:
 #### [Learn More about XmlTree Module](https://docs.python.org/3.7/library/xml.etree.elementtree.html#module-xml.etree.ElementTree)
 
 
+
+
 ## `Exploring JSON Module` exploring-json-module
 
 JSON is text, written with JavaScript object notation. JSON is a syntax for storing and exchanging data.
-
+It is commonly used for transmitting data in web applications (e.g., sending some data from the server to the client, so it can be displayed on a web page
+and vice versa
 - `json.loads()`
+-json. load() takes a file object and returns the json object. A JSON object contains data in the form of key/value pair. The keys are strings and the values are the JSON types
```

Comment:
Please remove the duplicate mention of json.load(), for e.g:

```suggestion
- `json.loads()` takes a file object and returns the json object. A JSON object contains data in the form of key/value pair. The keys are strings and the values are the JSON types
```

and remove duplicate json.dumps() in below lines as well.